src/App.vue

```diff
@@ -438,6 +438,12 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     this.$root.$off('go-to-dashboard')
   }
 
+  /** Navigates to current location, refreshing the page. */
+  protected reload (): void {
+    this.setHaveUnsavedChanges(false)
+    this.$router.go(0)
+  }
```

Comment:
Due to recent changes, while developing, some components don't reload correctly on hot-reloads. Now the retry button reloads the whole app, which takes of it.